### PR TITLE
Update SDK 1.0 to support schedule preview using periodic payment and organisation commission params

### PR DESCRIPTION
**Pull request by acostros**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<groupId>Mambu-APIs-Java</groupId>
 	<artifactId>Mambu-APIs-Java</artifactId>
 	<name>Mambu Java APIs SDK</name>
-	<version>9.4-bin</version>
+	<version>9.5-bin</version>
```
Comment (florinparascan) [inline pom.xml]: Should we delete 9.2? Also, in lib folder we have models starting from 8.0. Do we want to keep them all?

File: src/com/mambu/apisdk/model/ScheduleQueryParam.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package com.mambu.apisdk.model;
+
+/**
+ * Shows possible values of schedule query parameters
+ * 
+ * @author acostros
+ */
+public enum ScheduleQueryParam {
```
Comment (florinparascan) [inline src/com/mambu/apisdk/model/ScheduleQueryParam.java]: It looks like we also have some other parameters for repayment schedule, such as principalRepaymentInterval or repaymentPeriodCount. Is there any reason why we do not put the newly added params from this task in the same place with the other (old) ones?

File: pom.xml (hunk 1 above)
Comment (acostros) [reply pom.xml]: deleted them, thanks!

File: src/com/mambu/apisdk/model/ScheduleQueryParam.java (hunk 2 above)
Comment (acostros) [reply src/com/mambu/apisdk/model/ScheduleQueryParam.java]: added description on ScheduleQueryParams with the other params (loan related) that can be set

Comment (florinparascan) [reply src/com/mambu/apisdk/model/ScheduleQueryParam.java]: Thank you!

**Pull request merged**